udata/templates/dataset/card.html

```diff
@@ -1,3 +1,4 @@
+{% cache 60*5, 'dataset-card', dataset.id|string, g.lang_code %}
```

Comment:
At first, yes. It's already the case anyway ;) We can be more modular later.
Another option is dict + default value.